### ISSUE TITLE
SmartAmpPro (Pre-Release) not giving any good experience.

### ISSUE THREAD

**Issue by 38github** [GuitarML/SmartAmpPro#12]
I tried the Ubuntu vst3 placed in usual VST folder under OpenSUSE Tumbleweed but would not show up in Reaper. I then tried the windows vst3 in Windows 10 which showed up in Reaper but when I load any of the models available here it stops to respond.

**Comment by GuitarML** [comment]
@38github Thank you for the feedback, you are not the only one having issues with the pre release binaries. Do you get any sound from the Windows 10 VST in Reaper using the default model? And when you say "load any of the models" do you mean when you change the model selected in the dropdown, or when you use the "Import Tone" button? Or do you even see any model names in the drop down? Keep in mind that the SmartAmp Tonepack models are not compatible with the SmartAmpPro, even though they use the same .json extension. 

Some more info that may help troubleshoot: what audio interface are you using, and what version of Reaper are you using? 

When you go to "C:\Users\<your_username>\AppData\Roaming\GuitarML\SmartAmpPro\tones", you should see 5 .json files there, which get put there when you first load the VST3. If these files are missing, it may be a permissions problem and the files are not getting copied over.

**Comment by 38github** [comment]
[WIndows] The file was just a VST3 and no files with it so I copied it to my vst folder. I downloaded two compatible json files from this project and tried to load by choosing the button that says to load file or something. When the file was choosen and opened nothing happened and the plugin did not become responsive.

**Comment by GuitarML** [comment]
Right, so what's supposed to happen, is when the DAW scans the VST3, it actually copies 5 json files to the directory I mentioned from the VST3 binary data. This is supposed to give the user 5 default tones to use without having to train or import models. I have not done this in a plugin before, but it worked on my machine. Possibly a permissions issue, and since I built it on my machine it accepts it with no problems, but could give issues for other machines. What kind of device/audio interface are you using? Is it an ASIO device in Reaper/preferences/ device settings? Also I'm going to ask the go-to IT question.. have you restarted your computer and rerun Reaper?

**Comment by GuitarML** [comment]
For anyone running on Mac and not getting the SmartAmpPro running, you may need an additional step to allow the plugin to run, by telling the Mac to trust the developer of the plugin. Here are the steps:

https://support.allinlearning.com/hc/en-us/articles/200717214-Plugin-Installation-Error-on-Mac-can-t-be-opened-because-it-is-from-an-unidentified-developer-?mobile_site=true

I also recommend rebooting after copying the AU to the Components directory. As a developer, I can pay apple $100 a year to be a “trusted developer”, which is a future possibility but the project is not quite there yet.
